apps/services/auth/ids-api/infra/ids-api.ts

```diff
@@ -68,6 +68,11 @@ export const serviceSetup = (): ServiceBuilder<'services-auth-ids-api'> => {
         dev: '10001',
         staging: '6503760649',
       },
+      NOVA_ACCEPT_UNAUTHORIZED: {
+        dev: 'true',
+        staging: 'false',
+        prod: 'false',
```

Comment:
Let's add this to the other service too. (api? user profile?)

Reply:
Added configuration for service-portal-api, application-system-api and judicial-system-backend.